src/app/_api/data/nominations.ts

```diff
@@ -399,3 +399,49 @@ export const getNominationsCountForUser = async (
     { revalidate: CACHE_5_MINUTES },
   )();
 };
+
+export const getNominationsWeeklyStatsForUser = async (
+  user: User,
+): Promise<{
+  nominationsReceived: number;
+  nominationsSent: number;
+  pointsEarned: number;
+}> => {
+  return unstable_cache(
+    async () => {
+      const wallets = user.wallets.map((w) => w.wallet);
+      const [nominationsReceived, nominationsSent] = await Promise.all([
```

Comment:
I think this needs some tweaks.

When I'm nominating someone I send them Points and receive Points.
when someone nominates me they are sending me points and I will receive points.

so, nominations received we care about nominations where the destination_wallet_id is one of our wallets and we look at `boss_points_sent`

on nominations sent we care about nominations where the `origin_user_id` is our user_id (not wallet) and we care about `boss_points_received`

Reply:
Also I would test this with production data, because for example Jesse has a lot of nominations and I think that supabase caps the number of records returned at 1k. I think we can do count + sum on the same query tho! so that would fix the issue (and it is actually the only data we need)

Reply:
Yes indeed it's not clear to me how can I compute the points I earn when sending a nomination.